doc/sphinx/make-your-own-compositor.md

```diff
@@ -0,0 +1,258 @@
+---
+discourse: 4911,5164,5603,6756,8037
+---
+
+# Using Mir Libraries to Build Your Own Wayland Compositor
+If you've seen [the demo tutorial](explore-mir-using-demos), you're probably
+taken away by Mir's capabilities and want to use it and the supporting
+libraries to build your own Wayland compositor. This tutorial will guide you
+through the installation of dependencies, writing a simple compositor,
+building it, and running it. You'll be even more impressed by how easy it is to
+create a Wayland compositor using Mir!
+
+## Assumptions
+This tutorial assumes that:
+1. The reader is familiar with C++ and CMake.
+2. The reader has a C++ compiler with C++ 23 support.
```

Comment:
Mir/main needs C++23 support, but the tutorial doesn't and should "just work" with whatever compiler is on the reader's distro. (Because that's what that version of Mir was built with)